src/program.js

```diff
@@ -123,11 +125,22 @@ export class Program {
   }
 }
 
+// A global variable generated by DefinePlugin, generated in webpack.config.js
+declare var WEBEXT_BUILD_ENV: string;
 
-export function defaultVersionGetter(absolutePackageDir: string): string {
-  let packageData: any = readFileSync(
-    path.join(absolutePackageDir, 'package.json'));
-  return JSON.parse(packageData).version;
+export function defaultVersionGetter(
+  absolutePackageDir: string,
+  {localEnv: string = WEBEXT_BUILD_ENV} = {}
```

Comment:
Oh, sorry, I gave you the wrong example code. The lint error is correct and it needs to be fixed. The problem is actually an obscure ES6 vs Flow problem so let me offer the solution to help you move forward.

First, define a type for the option object that needs to be passed to `defaultVersionGetter()`, like this:

````javascript
type versionGetterOptions = {
  localEnv?: string,
};
````
Next, fix the definition of `defaultVersionGetter` to reference the new type:
````javascript
export function defaultVersionGetter(
  absolutePackageDir: string,
  {localEnv = WEBEXT_BUILD_ENV}: versionGetterOptions = {}
): string {
````
After this change, you'll need to update all the calls to `defaultVersionGetter()`. The flow checks in `npm start` should help guide you on what to change and where.